pgcli/main.py

```diff
@@ -103,16 +103,13 @@ def cli(database, user, password, host, port):
             try:
                 _logger.debug('sql: %r', document.text)
                 res = pgexecute.run(document.text)
-                output = []
```

Comment:
Ok, obviously missed that case.

It still seems suspicious to me if the `click.echo_via_pager('\n'.join(output))` line is correct, as old data is never flushed from the output but the pager is called for each such list.

Will test tomorrow and reset the code here if needed.